packages/docusaurus/package.json

```diff
@@ -48,6 +48,7 @@
     "@docusaurus/utils": "2.0.0-beta.18",
     "@docusaurus/utils-common": "2.0.0-beta.18",
     "@docusaurus/utils-validation": "2.0.0-beta.18",
+    "@reach/portal": "^0.16.2",
```

Comment:
Do we necessarily want to use an external dep? We'd rather implement a simple portal ourselves, maybe even exposing it as a core client API (as we did for `ErrorBoundary`), so that (a) we don't get blocked on an external dep (b) we can control bundle size and remove code that we don't need. Can we make our own abstraction over `React.createPortal`?

Reply:
I have updated it. Please have a look